server.js

```diff
@@ -18,6 +19,18 @@ app.get('/', function (req, res) {
   res.render('home')
 })
 
+app.get('/updated', function (req, res) {
+  var terminal = process.spawn('bash')
+  terminal.stdout.on('data', function (data) {
+    data = data.toString()
+    var ary = data.split('\n')
+    var lastUpdate = ary[1].split(',')[1]
+    res.send("{\"lastUpdate\":\""+lastUpdate+"\"}")
```

Comment:
Create a javascript object first and then use `JSON.stringify` to serialise it, like for the other endpoints